vagrant/provisioning/roles/ldap/tasks/main.yml

```diff
@@ -57,7 +57,24 @@
   loop:
     - "{{ ldap_group_base }}"
     - "{{ ldap_user_base }}"
-    
+
+- name: add portal organizational units
+  ldap_entry:
+    server_uri: "{{ ldap_url }}"
+    validate_certs: no
+    bind_dn: "{{ ldap_bind_user }}"
+    bind_pw: "{{ ldap_bind_password }}"
+    dn: "{{ item }}"
+    objectClass:
+      - organizationalUnit
+      - top
+    attributes:
+      description: "{{ item }}"
+  loop:
+    - "{{ ldap_portal_group_no_base }},{{ ldap_base }}"
```

Comment:
This block will work, but you could also just add these two items to the existing block that already adds OUs. This block isn't actually necessary.